forge-gui/res/cardsfolder/upcoming/oblivious_bookworm.txt

```diff
@@ -2,15 +2,15 @@ Name:Oblivious Bookworm
 ManaCost:G U
 Types:Creature Human Wizard
 PT:2/3
-T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ At the beginning of your end step, you may draw a card. If you do, discard a card unless a permanent entered the battlefield face down under your control this turn or you turned a permanent face up this turn.
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | OptionalDecider$ You | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ At the beginning of your end step, you may draw a card. If you do, discard a card unless a permanent entered the battlefield face down under your control this turn or you turned a permanent face up this turn.
 SVar:TrigDraw:DB$ Draw | RememberDrawn$ True | Optional$ True | SubAbility$ DBDiscard
```

Comment:
nah the Optional needs to happen on the effect line for Draw because of rules check
It was just named wrong, so change it to OptionalDecider instead